frontend/.eslintrc

```diff
@@ -21,11 +21,16 @@
       "jsx": true
     },
     "ecmaVersion": 2017,
-    "sourceType": "module"
+    "extraFileExtensions": [".json"],
+    "sourceType": "module",
+    "tsconfigRootDir": "./"
   },
   "plugins": ["react", "react-hooks", "@typescript-eslint"],
   "rules": {
-    "camelcase": 2,
+    "camelcase": [
+      "error",
+      { "allow": ["UNSAFE_componentWillReceiveProps", "UNSAFE_componentWillMount"] }
```

Comment:
@benjaminapetersen We should open a tech debt issue to track removing these

https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html